lib/gui/os/dialog/services/dialog.js

```diff
@@ -149,19 +159,19 @@ module.exports = function($q, SupportedFormatsModel) {
    * OSDialogService.showError('Foo Bar', 'An error happened!');
    */
   this.showError = (error, description) => {
```

Comment:
I guess this could use the same `this.showError = (error, description) => {` as https://github.com/resin-io/etcher/pull/1149#discussion_r104005634 ?